src/start/PasswordAuthScreen.js

```diff
@@ -90,38 +86,36 @@ class PasswordAuthView extends PureComponent<Props, State> {
 
     return (
       <Screen title="Log in" centerContent keyboardShouldPersistTaps="always" padding>
-        <View style={componentStyles.container}>
```

Comment:
This last commit is doing enough stuff that it'd be helpful to break it up.

Especially because one of the things it does is remove this wrapper, which changes the indentation of a bunch of code -- that makes the diff look considerably more complex than it is, so it'd be helpful to separate that from any other changes that aren't closely linked to it. (Which I think means it should go with just the `container.paddingBottom` hunk, and the rest should be separate?)

The other changes are a couple of different things, but because they're all small it's fine for them to be one commit.